chore: info about dedicated postgres instance (#5480)
influxdata/docs-v2: content/influxdb/clustered/install/prerequisites.md

- **PostgreSQL-compatible database** _(AWS Aurora, hosted Postgres, etc.)_:
  Used to store the InfluxDB catalog
  - Supported PostgreSQL versions: **13.8–14.6**

>   - Ensure that the PostgreSQL instance is dedicated exclusively to InfluxDB purposes to avoid conflicts. This prevents issues due to shared usage with other applications.

Reviewer: ```suggestion
  - Ensure that the PostgreSQL-compatible instance is dedicated exclusively to InfluxDB. This avoids conflicts and prevents issues caused by shared usage with other applications.
```

Author: done!

Running the Catalog database and Object Store in a separate namespace or outside
of Kubernetes makes management of the InfluxDB instance easier and helps to
prevents accidental data loss.

> While deploying everything in the same namespace is possible for testing or initial setups, it is not recommended for production environments.

Reviewer: ```suggestion
While deploying everything in the same namespace is possible for testing or initial setup, it is not recommended for production environments.
```
